scanner/k8s-assets/processor/config.go

```diff
@@ -0,0 +1,10 @@
+// SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Greenhouse contributors
+// SPDX-License-Identifier: Apache-2.0
+
+package processor
+
+type Config struct {
```

Comment:
Ahh here is the Cluster and Region!

I wonder myself though if the Processor should have this as an environment variable. This strongly couples the processor to be in the same region as the scanner in future. Ideally, we can later have 1 processor and 10 scanners for this at some point in time. Meaning the scanner is the actually part that is "region" and "cluster" aware.

Reply:
Fine for me for the PoC though.